### PR TITLE
Add description of PowerShell Core configuration settings

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

The document describe PowerShell Core configuration settings, polices and options.

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [NA] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [NA] Issue filed - Issue link:
- [NA] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [NA] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [NA] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

File: docs/learning-powershell/PowerShellConfigurationSettings.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Priority of applying settings
+
+Because a configuration setting can be in several schemes, the setting wins according to the priority of its scheme.
+
+### Priorities for Policies in descending order
+
+Scheme | Windows | Unix
+-| - | -
+GPO -> Computer Policy | HKLM\Software\Policies\Microsoft\Windows\PowerShellCore | /etc/powershell.config.json
```
Comment (daxian-dbw) [inline docs/learning-powershell/PowerShellConfigurationSettings.md]: The PowerShell Core GPO registry keys are at `Software\Policies\Microsoft\PowerShellCore`.

**Comment by daxian-dbw** [comment]
The configuration file locations and scopes in the doc don't match with what we have today. Today we have two scopes: system-wide (config file is located at `$pshome`) and current-user-wide (config file is located at the same folder as `$profile`).
If you are proposing the way you think configuration should be arranged, please use a RFC instead of documentation.

File: docs/learning-powershell/PowerShellConfigurationSettings.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+GPO -> User Policy | HKCU\Software\Policies\Microsoft\Windows\PowerShellCore |
+File -> Computer-Wide | %ProgramFiles%/PowerShell/powershell.config.json | /opt/Microsoft/powershell/powershell.config.json
```
Comment (dantraMSFT) [inline docs/learning-powershell/PowerShellConfigurationSettings.md]: This should refer to $PSHOME versus the explicit path.

**Comment by dantraMSFT** [comment]
Shouldn't this be in an about md file?

(powershell-docs/reference/6/Microsoft.PowerShell.Core/About)

**Comment by SteveL-MSFT** [comment]
@iSazonov are you documenting the current behavior or trying to define desired behavior?  If the latter, I agree with @daxian-dbw that this should be a RFC

**Comment by iSazonov** [comment]
Moved to https://github.com/PowerShell/PowerShell-RFC/pull/111

**Pull request closed without merging**